SER515Project_Team3/WebContent/student1.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Student | Grade 1</title>
+  <script src="./lib/blockly-master/blockly_compressed.js"></script>
+  <script src="./lib/blockly-master/blocks_compressed.js"></script>
+  <script src="./lib/blockly-master/msg/js/en.js"></script>
+  <style>
```

Comment:
Can we add this in a .css file under css directory?

Reply:
This is possible.